BuildaGitServer/GitServerPublic.swift

```diff
@@ -48,6 +53,7 @@ public enum GitService: String {
     public func accessTokenUrl() -> String {
         switch self {
         case .GitHub: return "https://github.com/login/oauth/access_token"
+        case .EnterpriseGitHub: return "https://\(hostname())/login/oauth/access_token"
```

Comment:
I think we'll disallow the classic OAuth flow for Enterprise for now, so feel free to add an assert here (when GitService is EnterpriseGitHub, it should never be asked for accessTokenUrl)